.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Please delete. .vscode should be in your .gitignore file ;-)

Reply:
no worries - added to .gitignore.
(was using vscode - github codespaces :D)